mobile/src/pages/node/node.ts

```diff
@@ -106,55 +125,40 @@ export class NodePage {
       channel: this.node.channel_id
     };
 
-    this.storage.get("history").then((val) => {
-      if(val != null && val != '' && val && val.length > 0) {
-        if(val.filter(item => item.channel_id === this.node.channel_id).length != 0){
-          
-        } else {
-          loader.present();
-        }
-      }
-    });
+    loader.present();
 
     this.http.post(this.history_node_api, params).subscribe((result: any) => {
       console.log(result);
 
       if(result.success == '100' && result.feed.hourly_results){
         if(result.feed.hourly_results.length > 0){
           this.history_node_api_success = true;
-          this.node.refreshed     = result.feed.hourly_results[0].time;
-          this.node.feeds.field1  = result.feed.hourly_results[0].pm2_5;
+          // this.node.time     = result.feed.hourly_results[0].time;
+          // this.node.value  = result.feed.hourly_results[0].pm2_5;
 
           this.offlineStoreHistoryStoreNodeInfo(result.feed.hourly_results, this.node);
           this.getHistoryGraphData(result.feed.hourly_results);
         }
         loader.dismiss();
       } else {
+        loader.dismiss();
 
         this.toastCtrl.create({
-          message: 'History information not available',
+          message: 'Up-to-date history information unavailable',
```

Comment:
Just curious about how we can minimise this situation of not having up to date historical values. cc: @sserurich